modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/Header/index.js

```diff
@@ -33,6 +35,18 @@ const ActivationKeysTableHeader = ({
 }) => {
 	const [activationKeys, setActivationKeys] = activationKeysState;
 
+	const {
+		userAccountsState: [userAccounts],
+	} = useGetAccountUserAccount(project);
+
+	const isAdminOrPartnerManager = userAccounts?.some((userAccount) =>
+		userAccount?.roles?.some(
```

Comment:
Oi Luisa, aqui dá pra usar somente um some e comparar com userAccoun.role.
```suggestion
		userAccount?.some((user) => user?.role === ROLE_TYPES.admin.key || ROLE_TYPES.partnerManager.key)
```

Reply:
userAccounts retorna todos os usuários vinculados àquela conta, certo? O ideal é que exista um filtro pra encontrar o usuário logado e a partir dai ver se ele é admin ou manager.
```suggestion
		userAccounts?find(({id}) => id === +Liferay.ThemeDisplay.getUserId());
```